source/GameInterface.Tests/Serialization/SerializerTests/EquipmentElementSerializationTest.cs

```diff
@@ -0,0 +1,69 @@
+﻿using GameInterface.Serialization.Impl;
+using GameInterface.Serialization;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TaleWorlds.Core;
+using Xunit;
+using TaleWorlds.ObjectSystem;
+using System.Reflection;
+
+namespace GameInterface.Tests.Serialization.SerializerTests
+{
+    public class EquipmentElementSerializationTest
+    {
+        [Fact]
+        public void EquipmentElement_Serialize()
+        {
+            EquipmentElement equipmentElement = new EquipmentElement();
+
+            BinaryPackageFactory factory = new BinaryPackageFactory();
+            EquipmentElementBinaryPackage package = new EquipmentElementBinaryPackage(equipmentElement, factory);
+
+            package.Pack();
+
+            byte[] bytes = BinaryFormatterSerializer.Serialize(package);
+
+            Assert.NotEmpty(bytes);
+        }
+
+        [Fact]
+        public void EquipmentElement_Full_Serialization()
+        {
+            MBObjectManager.Init();
```

Comment:
Create a constructor for EquipmentElementSerializationTest and move this line there. I didn't think to test it this way, good idea